File: Coding/daytwob.cpp
#include<iostream>
using namespace std;
int main(){
    int a;
    cout<<"Enter your marks:";
    cin>>a;
    if(a>=90)
    cout<<"A";
    else if(a>=70)
    cout<<"B";
    else if(a>=50)
    cout<<"C";
    else
    cout<<"Fail";
}

File: Coding/daytwob.py
# Take marks → print grade:
# 90+ → A
# 70+ → B
# 50+ → C
# else → Fail
a=int(input("Enter your marks:"))
if a>=90:
    print("A")
elif a>=70:
    print("B")
elif a>=50:
    print("C")
else:
    print("Fail")